python/llm/src/ipex_llm/transformers/npu_model.py

```diff
@@ -259,6 +259,7 @@ def optimize_npu_model(cls, *args, **kwargs):
                 import convert_llm
             convert_llm(llm,
                         kv_len=max_output_len,
+                        max_prompt_len=max_prompt_len,
```

Comment:
`max_prompt_len` should be smaller than 4096 in the current implementation?

Reply:
I think in the current implementation, `max_prompt_len` should be smaller than `kv_len` (like 1023) ?

Reply:
In the cpp implementation, the prompt buffer is set to 4096; and is there a constraint of `kv_len` or `max_output_len`?

Reply:
In the latest implementation, prompt buffer is not used now.
There is no constraint of `kv_len` or` max_output_len`, but has constraint for new tokens now as limitation of `Init_pipe()`.
Added a check for this first.